components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -256,8 +281,24 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
         return this.targetRef.current;
     }
 
-    private getLastStep(tutorialSteps: Record<string, number>) {
-        return Object.values(tutorialSteps).reduce((maxStep, candidateMaxStep) => {
+    private getLastStep(tutorialSteps: Record<string, number>, category: string) {
+        let steps = tutorialSteps;
+
+        // temporary fix for filtering tutorial_step category based on role
+        // however this could work if TutorialSteps are converted to an enum and tips for only admins added last to the enum
+        if (category === Preferences.TUTORIAL_STEP) {
```

Comment:
thinking this better, if the idea is just remove the preference `START_TRIAL` property and value from the steps object, I think that  just check for the condition if the user is admin and then delete the property from the object with `delete Obj.property`, WDYT @AGMETEOR ?

Reply:
That's much better. Thanks :)